vms/event/models.py

```diff
@@ -1,67 +1,68 @@
-                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)|(\')]+$',
-            ),
-        ],
-    )
-    start_date = models.DateField()
-    end_date = models.DateField()
-
-    address = models.CharField(
-        max_length=30,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\')]+$',
-            ),
-        ],
-        blank=True,
-        null=True,
-    )
-    city = models.CharField(
-        max_length=30,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$',
-            ),
-        ],
-        blank=True,
-        null=True,
-    )
-    state = models.CharField(
-        max_length=30,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\-)]+$',
-            ),
-        ],
-        blank=True,
-        null=True,
-    )
-    country = models.CharField(
-        max_length=30,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$',
-            ),
-        ],
-        blank=True,
-        null=True,
-    )
-
-    venue = models.CharField(
-        max_length=30,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$',
-            ),
-        ],
-        blank=True,
-        null=True,
-    )
+from django.core.validators import RegexValidator
+from django.db import models
+
+
+class Event(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(
+        max_length=75,
+        validators=[
+            RegexValidator(
+                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)|(\')]+$',
+            ),
+        ],
+    )
+    start_date = models.DateField()
+    end_date = models.DateField()
+
+    address = models.CharField(
+        max_length=30,
+        validators=[
+            RegexValidator(
+                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\')]+$',
+            ),
+        ],
+        blank=True,
+        null=True,
+    )
+    city = models.CharField(
+        max_length=30,
+        validators=[
+            RegexValidator(
+                r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$',
+            ),
+        ],
+        blank=True,
+        null=True,
+    )
+    state = models.CharField(
+        max_length=30,
+        validators=[
+            RegexValidator(
+                r'^[(A-Z)|(a-z)|(\s)|(\-)]+$',
+            ),
+        ],
+        blank=True,
+        null=True,
+    )
+    country = models.CharField(
+        max_length=30,
+        validators=[
+            RegexValidator(
+                r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$',
+            ),
+        ],
+        blank=True,
+        null=True,
+    )
+
+    venue = models.CharField(
+        max_length=30,
+        validators=[
+            RegexValidator(
+                r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$',
+            ),
+        ],
+        blank=True,
+        null=True,
```

Comment:
@amruthasangeeth Complete file is coming as diff, it is really difficult to review. Can you please undo the unnecessary changes (cleanups, spacing)?